eth/bor_api_backend.go

```diff
@@ -74,8 +75,9 @@ func (b *EthAPIBackend) GetVoteOnHash(ctx context.Context, starBlockNr uint64, e
 	isLocked := downloader.LockMutex(endBlockNr)
 
 	if !isLocked {
-		downloader.UnlockMutex(false, "", endBlockNr, common.Hash{})
-		return false, errors.New("whitelisted number or locked sprint number is more than the received end block number")
+		// downloader.UnlockMutex(false, "", endBlockNr, common.Hash{})
```

Comment:
Can we clarify why those two lines are commented out?

Reply:
Added.